WordPress/Classes/Extensions/UIColor+MurielColors.swift

```diff
@@ -247,4 +247,34 @@ extension UIColor {
             return WPStyleGuide.greyLighten10()
         }
     }
+
+    /// MARK: Muriel colors for buttons
+    static var buttonBase: UIColor {
```

Comment:
Should we give these types of button a specific name? There may be more types of button! `primaryButton`? Or do we address that if necessary later?

Reply:
I'm not sure on this one!